### PR TITLE
Brainredux

### PR DESCRIPTION
**Pull request by kescobo**

### CODE REVIEW

**Comment by kescobo** [comment]
@Hugemiler Main thing here is that I changed the `load(::Neuroimaging)` to generate a `CommunityProfile` a la Microbiome.jl

```
julia> brain = Resonance.load(Resonance.Neuroimaging())
CommunityProfile{Float64, BrainVolume, MicrobiomeSample} with 99 features in 1179 samples

Feature names:
lateral-ventricle, inferior-lateral-ventricle, cerebellum-exterior...White-matter, has_segmentation

Sample names:
sub1_tp2, sub2_tp1, sub3_tp2...sub996_tp3, FG00857



julia> first(features(brain), 3)
3-element Vector{BrainVolume}:
 BrainVolume("lateral-ventricle", Resonance.Left())
 BrainVolume("inferior-lateral-ventricle", Resonance.Left())
 BrainVolume("cerebellum-exterior", Resonance.Left())

julia> first(samples(brain), 3)
3-element Vector{MicrobiomeSample}:
 MicrobiomeSample("sub1_tp2", {:subject = 1, :timepoint = 2, :ageMonths = 23.638356164383563, :sex = "Male", :hassample = false})
 MicrobiomeSample("sub2_tp1", {:subject = 2, :timepoint = 1, :ageMonths = 3.978082191780822, :sex = "Male", :hassample = false})
 MicrobiomeSample("sub3_tp2", {:subject = 3, :timepoint = 2, :ageMonths = 132.8876712328767, :sex = "Female", :hassample = false})
```

I set it up so that scans that don't have associated stools samples get named eg `"sub1_tp2"`, and get the metadata that comes in the brain table, but if they have stool samples, they get named appropriately. You can even pass `; samplefield="etoh"` to `load` if you want to get the associated ethanol sample (for working with metabolomics).

File: src/data_loading.jl [hunk 1]
```diff
@@ -1,3 +1,68 @@
...
+function brainvolume(s::String)
+    if startswith(s, r"(left|right)-")
+        parts = split(s, '-')
+        BrainVolume(join(parts[2:end], '-'), Symbol(first(parts)))
+    else
+        BrainVolume(s, missing)
+    end
+end
+
+@testset "Brain Volumes" begin
```
Comment (Hugemiler) [inline src/data_loading.jl:47]: :clap:

File: data/timepoints_final.csv [hunk 2]
```diff
@@ -1,1716 +1,1716 @@
-subject,timepoint,ageMonths,mother_HHS_Education,race,cogScore,has_segmentation,ECHOTPCoded,assessmentDate,scanDate,ed,rce,lhCorticalWhiteMatterVol,lhCortexVol,Left-Thalamus,Left-Lateral-Ventricle,Left-Cerebellum-White-Matter,Left-Cerebellum-Cortex,Left-Caudate,Left-Putamen,Left-Pallidum,Left-Hippocampus,Left-Amygdala,Left-Accumbens-area,Left-VentralDC,Left-choroid-plexus,rhCorticalWhiteMatterVol,rhCortexVol,Right-Thalamus,Right-Lateral-Ventricle,Right-Cerebellum-White-Matter,Right-Cerebellum-Cortex,Right-Caudate,Right-Putamen,Right-Pallidum,Right-Hippocampus,Right-Amygdala,Right-Accumbens-area,Right-VentralDC,Right-choroid-plexus,Brain-Stem,CSF,EstimatedTotalIntraCranialVol,has_cogScore,has_stool,has_metabolomics,has_prevstool,has_futstool,has_scan,keeper,omni,etoh
```
Comment (Hugemiler) [inline data/timepoints_final.csv:1]: Do I need to check that the new table is coherent with the old one? Or did you just remove columns and thats it?

Comment (kescobo) [reply data/timepoints_final.csv:1]: It's mostly coherant. I changed `rce` to `race` and `ed` to education, and made them both proper categorical variables (ordinal for the later, [see here](https://github.com/Klepac-Ceraj-Lab/Resonance/pull/70/files/f4f87bf66b5b4720e381b1da41c261e02bf74e68#diff-fbbff7ed0488dd4abea9e613edb9d64876bc7c40348f7a8aca69cc073e1e28bfR111)), plus removed all of the brain columns.

**Pull request merged**